pkg/common/connectionmanager/search.go

```diff
@@ -202,12 +202,14 @@ func (cm *ConnectionManager) WhichVCandDCByNodeID(ctx context.Context, nodeID st
 					hostName = myNodeID
 				}
 
+				UUID := strings.ToLower(strings.TrimSpace(oVM.Summary.Config.Uuid))
+
 				klog.V(2).Infof("Found node %s as vm=%+v in vc=%s and datacenter=%s",
 					nodeID, vm, res.vc, res.datacenter.Name())
-				klog.V(2).Info("Hostname: ", hostName, " UUID: ", oVM.Summary.Config.Uuid)
+				klog.V(2).Info("Hostname: ", hostName, " UUID: ", UUID)
```

Comment:
same

Reply:
Same as above

Reply:
Just a nit, I like using printf style formatting instead of appending variables, it would make it consistent with the rest of the code.

Reply:
Yup. Updated for uniformity.